docs/pages/accounts/sso.mdx

```diff
@@ -56,15 +54,17 @@ SSO users are like regular users. However, there are a few known exceptions:
 
 ### Transitioning existing users to SSO
+
 Regular users may be a member of one or many personal, team, and organization accounts while SSO users belong exclusively to their organization account. Thus, existing users cannot be directly converted into SSO users. However, a regular user that's already a member of your organization may create a second user by going to the [SSO login page](https://expo.dev/sso-login). Then, their regular user can be removed from the organization.
 
 To transition from using a regular Expo account to an SSO account, follow these steps:
+
 1. Check if you're already logged in at [expo.dev](https://expo.dev). If so, log out.
 2. Go to the [SSO login page](https://expo.dev/sso-login) and follow the prompts, such as entering your organization name, creating a new Expo username, and logging in to your identity provider.
-2. By default, your new SSO user will have the View Only role. If you need a different role, ask an Admin or Owner to update your role in [member settings](https://expo.dev/accounts/[account]/settings/members).
-3. Run `eas login --sso` to switch to your new account on the CLI.
-4. At this time, the Admin or Owner can remove your old user from the organization. In [member settings](https://expo.dev/accounts/[account]/settings/members), the list of organization members indicates whether a user is an SSO or non-SSO user. The Admin or Owner can click the dropdown next to the old user and click **Remove member**.
-5. If you no longer need your old user account, log out of your new SSO account, then log in to your old account and go to [user settings](https://expo.dev/settings). Scroll down and click **Delete Account**. **Note that this will delete any projects under your old user account.** It will not affect any projects owned by the organization.
+3. By default, your new SSO user will have the View Only role. If you need a different role, ask an Admin or Owner to update your role in [member settings](https://expo.dev/accounts/[account]/settings/members).
+4. Run `eas login --sso` to switch to your new account on the CLI.
+5. At this time, the Admin or Owner can remove your old user from the organization. In [member settings](https://expo.dev/accounts/[account]/settings/members), the list of organization members indicates whether a user is an SSO or non-SSO user. The Admin or Owner can click the dropdown next to the old user and click **Remove member**.
+6. If you no longer need your old user account, log out of your new SSO account, then log in to your old account and go to [user settings](https://expo.dev/settings). Scroll down and click **Delete Account**. **Note that this will delete any projects under your old user account.** It will not affect any projects owned by the organization.
```

Comment:
Thanks for correcting this! We also could use `Step` component here in the future.

Reply:
Great catch! Let me update the PR.

Reply:
Formatted the whole guide to use `Step` component and also find another issue where an internal link was missing.